### PR TITLE
Simplified configuration

### PR DESCRIPTION
**Pull request by mortezaalizadeh**

### CODE REVIEW

File: src/exchange/docker-compose.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ services:
       target: local
     restart: on-failure
     environment:
-      OMIGA_ENVIRONMENT: dockercompose
+      OMIGA_POSTGRES_CONNECTIONSTRING: "postgres://postgres:12345678@postgres/exchange"
```
[the remark this answers is not in the document]
Comment (mortezaalizadeh) [reply src/exchange/docker-compose.yml:23]: it is on docker compose file, not in the code. How else would you like to do that? @sepisoad

[the remark this answers is not in the document]
Comment (sepisoad) [reply src/exchange/docker-compose.yml:23]: env variables!?

[the remark this answers is not in the document]
Comment (mortezaalizadeh) [reply src/exchange/docker-compose.yml:23]: that's what I am doing

**Review by sepisoad — APPROVED** [review]
kha

**Pull request merged**